doc/multi-components.rst

```diff
@@ -0,0 +1,48 @@
+Haddocks of multiple components
+===============================
+
+Haddock supports building documentation of multiple components.  First, one
+needs to build haddocks of all components which can be done with:
+
+.. code-block:: none
+
+    cabal haddock --haddock-html \
+                  --haddock-quickjump \
+                  --haddock-option="--use-index=../doc-index.html" \
+                  --haddock-option="--use-contents=../index.html" \
+                  --haddock-option="--base-url=.." \
+                  all
+
+The new `--base-url` option will allow to access the static files from the
```

Comment:
I believe that should rather be
```
``--base-url``
```

Similarly below.

Reply:
You're right, it's not markdown :).

Reply:
Fixed.